diff -urN qingy-0.2.3/media/default/Makefile.in qingy-0.2.3.patch/media/default/Makefile.in
--- qingy-0.2.3/media/default/Makefile.in	2003-10-21 10:51:18.000000000 +0200
+++ qingy-0.2.3.patch/media/default/Makefile.in	2003-12-28 23:23:20.000000000 +0100
@@ -80,7 +80,7 @@
 am__quote = @am__quote@
 install_sh = @install_sh@
 
-defaultthemedatadir = $(prefix)/etc/qingy/themes/default
+defaultthemedatadir = $(datadir)/qingy/themes/default
 
 defaultthemedata_DATA = \
 	theme                 \
diff -urN qingy-0.2.3/media/Makefile.in qingy-0.2.3.patch/media/Makefile.in
--- qingy-0.2.3/media/Makefile.in	2003-10-21 10:51:18.000000000 +0200
+++ qingy-0.2.3.patch/media/Makefile.in	2003-12-28 23:05:52.000000000 +0100
@@ -82,10 +82,10 @@
 
 SUBDIRS = default sessions @PAM@
 
-settingsdatadir = $(prefix)/etc/qingy
+settingsdatadir = /etc/qingy
 settingsdata_DATA = settings
 
-directfbrcdir = $(prefix)/etc
+directfbrcdir = /etc
 directfbrc_DATA = directfbrc.qingy
 
 BUILT_SOURCES = directfbrc.qingy
diff -urN qingy-0.2.3/media/pam/Makefile.in qingy-0.2.3.patch/media/pam/Makefile.in
--- qingy-0.2.3/media/pam/Makefile.in	2003-10-21 10:51:19.000000000 +0200
+++ qingy-0.2.3.patch/media/pam/Makefile.in	2003-12-28 23:07:37.000000000 +0100
@@ -80,7 +80,7 @@
 am__quote = @am__quote@
 install_sh = @install_sh@
 
-defaultpamdatadir = $(prefix)/etc/pam.d
+defaultpamdatadir = /etc/pam.d
 
 defaultpamdata_DATA = \
 	qingy
diff -urN qingy-0.2.3/media/sessions/Makefile.in qingy-0.2.3.patch/media/sessions/Makefile.in
--- qingy-0.2.3/media/sessions/Makefile.in	2003-10-21 14:09:10.000000000 +0200
+++ qingy-0.2.3.patch/media/sessions/Makefile.in	2003-12-28 23:08:53.000000000 +0100
@@ -62,7 +62,7 @@
 OBJEXT = @OBJEXT@
 PATH_SEPARATOR = @PATH_SEPARATOR@
 
-bindir = $(prefix)/etc/qingy/sessions
+bindir = /etc/qingy/sessions
 AMTAR = @AMTAR@
 AWK = @AWK@
 CC = @CC@
